utils/src/main/kotlin/Utils.kt

```diff
@@ -66,6 +66,24 @@ val ROOT_LICENSE_FILENAMES = listOf(
     "readme*"
 ).generateCapitalizationVariants()
 
+/**
+ * The current user's home directory.
+ */
+val userHomeDirectory by lazy {
+    File(object {}.fixupUserHomeProperty())
```

Comment:
Btw, why is `fixupUserHomeProperty` an extension function? Maybe historic reasons? If I don't miss anything it could just be a top level function.

Reply:
It became an extension function recently as I wanted to call `log` inside of it. And I wanted to call `log` inside of it instead of in the caller as I also wanted to log the previous value of `user.home`.